users/auth.go

```diff
@@ -1,54 +1,6 @@
 package users
```

Comment:
Yeah, that's how it started shaking out. Once I finished re-arranging fields between UserAuth and UserPII, I ended up with the handful of auth-related functions that you now see in `auth.go`